src/Perpetuum/Services/RiftSystem/Rift.cs

```diff
@@ -194,13 +194,12 @@ public void UseItem(Player player)
 
 
             // we are on a stronghold. we want to go home.
-            // for the moment we will send them to a teleporter on zone 0.
+            // Sending them at the Hershfield main terminal.
             if (player.Zone is StrongHoldZone)
             {
-                var destZone = player.Character.GetZone(0);
-                var teleportColumns = destZone.GetTeleportColumns().Where(t => t.IsEnabled);
+                var destZone = player.Character.GetZone(8); // Hershfield zone
                 var teleport = _teleportStrategyFactories.TeleportToAnotherZoneFactory(destZone);
-                teleport.TargetPosition = teleportColumns.First().CurrentPosition;
+                teleport.TargetPosition = new Position(964, 804); // At the Hershfield main terminal
```

Comment:
Try using a RandomWalkablePosition around this point - it will more safely resolve a place a player may be able to walk out of, in case things about the zone/layers are changed.

Reply:
Actually scratch that: following loyals comments - find the DockingBase itself off of the zone directly, and call methods on it to resolve a position
Like this: https://github.com/OpenPerpetuum/PerpetuumServer/blob/Development/src/Perpetuum/Units/DockingBases/DockingBase.cs#L30

There are ways to lookup "units" on a zone by type - much like the teleport columns.  You will need to do that for DockingBases, and look for the first/only one that is available on zone 8

Reply:
I added a new commit but using DockingBaseHelper is adding a new dependency to the Rift class.